wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,75 @@
+
+## Oh, no! I can't access to my pc!
+
+1. Enter Safe Mode:
+
+Safe mode is a diagnostic boot way that loads Windows with only the essential drivers. This allows you to access Windows even if the problematic driver is causing issues during normal boot.
+
+ - Windows 10/11: Repeatedly press and hold the power button while booting to trigger a forced restart. After 2–3 restarts, you'll see the “Advanced options” screen. Navigate to “Troubleshoot” > “Advanced options” > “Startup Settings” > “Restart now”. Upon restart, choose “Enable Safe Mode with Networking” (if you need internet) or simply “Safe Mode”.
+
+ - Earlier Windows versions: Restart your computer and repeatedly press F8 before the Windows logo appears. This should open the Advanced Boot Options menu, where you can choose “Safe Mode”.
+
+2. Uninstall the Driver in Safe Mode:
+
+Once in safe mode, you can uninstall the problematic driver:
+
+ 1. Open the Device Manager (search for it in the Start Menu).
+ 2. Expand the category for the device with the recently installed driver.
+ 3. Right-click on the driver and select “Uninstall device”.
+ 4. Check the box “Delete the driver software for this device” (if available) and click “Uninstall”.
+
+3. System Restore (if applicable):
+
+If you recently created a system restore point before installing the driver, you can restore Windows to that point:
+
+ 1. In Safe Mode, open the Start Menu and search for “System Restore”.
+ 2. Click “Create a restore point” and then “Next”.
+ 3. Choose the desired restore point (ideally the one before installing the driver) and click “Next” followed by “Finish”.
+
+4. Startup Repair (if Windows won't boot at all):
+
+If even Safe Mode isn't accessible, you can try Startup Repair, a built-in tool on Windows that tries to fix boot issues:
+
+ 1. Create a Windows Recovery USB drive using another computer [https://msft.chat/wiki/downloading-windows.html](https://msft.chat/wiki/downloading-windows.html).
+ 2. Boot your computer using the recovery drive and choose “Fix computer problems > Troubleshoot” > “Advanced options” > “Startup Repair”.
+
+5. Reinstall Windows (last resort):
+
+If none of the above solutions work, you may need to reinstall Windows. This will erase all your data and applications, so make sure you have a backup.
+
+ 1. Download windows with a functional machine, following the steps from this URL [https://msft.chat/wiki/downloading-windows.html](https://msft.chat/wiki/downloading-windows.html)
+ 2. Next, follow this guide about how to install: [https://msft.chat/wiki/installing-windows.html](https://msft.chat/wiki/installing-windows.html) (if asks about a licence, and you don't have it, you can press Skip or "I don't have a product key" and Windows will reactivate automatically later. If you were previously signed in with a Microsoft account, you may need to sign in with the same account again).
```

Comment:
Next, to install Windows, follow this guide : [https://msft.chat/wiki/installing-windows.html](Instaling Windows) 

Note : During the installation process, you may be asked to enter a licence key. If you don't have one, you can click Skip or "I don't have a product key" and Windows will automatically reactivate later. However, you will need to sign-in with a Microsoft account.

Reply:
@sthivaios Your link syntax is backwards, it's `[text to display](url)`

You also do not always have to sign in with a MS account for Windows to find a digital license, though that will probably improve your chances of successful reactivation

Reply:
> @sthivaios Your link syntax is backwards, it's `[text to display](url)` (check your other suggestions too)
> 
> You also do not always have to sign in with a MS account for Windows to find a digital license, though that will probably improve your chances of successful reactivation

wait really? 💀 😭

Reply:
Heh, then what I do?

Reply:
@sthivaios re:
> However, you may need to sign-in with a Microsoft account.

Personally I think that what was there before:
> If you were previously signed in with a Microsoft account, you may need to sign in with the same account again

...fits better, because it makes it clear that you need to use the same account if the key is tied to your MS account.

Reply:
> @sthivaios re:
> 
> > However, you may need to sign-in with a Microsoft account.
> 
> Personally I think that what was there before:
> 
> > If you were previously signed in with a Microsoft account, you may need to sign in with the same account again
> 
> ...fits better, because it makes it clear that you need to use the same account if the key is tied to your MS account.

oh yeah that's better